third-party/realdds/doc/initialization.md

```diff
@@ -48,14 +48,28 @@ This is the very first message, for example:
 }
 ```
 
-Mainly the number of streams to expect. The device will wait for this many stream headers to arrive to finish initialization.
+- `n-streams` is the number of streams to expect
+     The device will wait for this many stream headers to arrive to finish initialization
+- `extrinsics` describe world coordinate transformations between any two streams in the device, required for proper translation of pixel coordinates between sensors, such as when a point-cloud is needed
+- `presets` is an optional array of preset names
+    The presets may then be applied using `change-preset`
 
-The `extrinsics` describe world coordinate transformations between any two streams in the device. This is required for proper alignment on the client.
+#### Extrinsics
 
-Optionally, the device may supply a `presets` array of preset names. The presets may then be applied (currently not implemented).
+In order to translate from one stream viewpoint to another, a graph needs to be available with nodes for each stream and a directional edge between them (one from A to B; another from B to A).
 
+So `extrinsics` is an array of such edges: `[<edge1>,<edge2>,...]`,
+Each `edge` is also an array: `[<from>,<to>,[<r00>,<r01>,...,<r22>,<tx>,<ty>,<tz>]]`.
+Where `<from>` and `<to>` are stream names, and the array inside contains the 9 rotation values (column-major) followed by 3 translation vector values.
 
-#### `device-options`
+The whole graph can be transmitted like this but this is overkill: given at least one edge from each stream to another, the rest can be computed. This is what librealsense does:
```

Comment:
I think the term is "complete" graph when all nodes are connected to all other nodes.

Reply:
OK, will change.